domain/src/main/java/com/architect/codes/mu8/callback/ComicRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.architect.codes.mu8.callback
+
+import com.architect.codes.mu8.model.Comic
+
+open interface ComicRepository {
+    suspend fun getAllComics(): List<Comic>
```

Comment:
If you make this also an operator, instead of calling `getAllComics()` you can just call `comicRepository()`, check `CharactersUseCase` file for an example.